src/test/java/cn/edu/tsinghua/tsfile/timeseries/read/QueryEnginePerf.java

```diff
@@ -193,6 +193,7 @@ static public void writeToFile(FileSchema schema) throws InterruptedException, I
 			innerWriter.write(record);
 			lineCount++;
 		}
+		in.close();

```

Comment:
Considering time cost testing, it's better to move `in.close()` after the next line, `endTime ...`

Reply:
move `in.close` after the endTime